lib/testLauncher/commands/automated.js

```diff
@@ -47,20 +47,20 @@ const builder = yargs => {
 };
 
 const handler = async(argv) => {
-	const userConfig = argv.configFile ? readUserConfig(argv.configFile) : {};
-	const ownArgs = mergeConfigs(mergeConfigs(launcherParams, userConfig), argv);
+	const {config, userCommandArgs} = await composeConfig(argv, false);
 
-	override(ownArgs);
+	override(config);
 
-	const userCommandArgs = hideOwnArgs();
-	const suitestLauncher = new SuitestLauncher(ownArgs, userCommandArgs);
+	const suitestLauncher = new SuitestLauncher(config, userCommandArgs);
 
 	await suitestLauncher.runAutomatedSession();
 };
 
+/* istanbul ignore next */
 module.exports = {
 	command,
 	describe,
 	builder,
-	handler,
+	// Make sure handler is called only once (yargs bug)
```

Comment:
@givehug , so is it confirmed to be a bug or an assumption? If available, please put here a link to corresponding issue in their repo.

Reply:
https://github.com/yargs/yargs/issues/1069#issuecomment-468451450